src/DevBetterWeb.Core/Specs/MemberByUserIdWithBooksReadSpec.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Ardalis.Specification;
+using DevBetterWeb.Core.Entities;
+
+namespace DevBetterWeb.Core.Specs
+{
+    public class MemberByUserIdWithBooksReadSpec : Specification<Member>
+    {
+        public string UserId { get; }
+
+        public MemberByUserIdWithBooksReadSpec(string userId)
+        {
+            UserId = userId;
```

Comment:
What is this property used for?

Reply:
it's used to make sure you get the member with the userId you pass in

Reply:
removed